### DISCUSSION TITLE
Setting tween resolution values for all instances of phantomcamera2D

### DISCUSSION THREAD

**Discussion by 2yellow4u** [ramokz/phantom-camera#349] [Question / Help Wanted] [answered]
Sorry in advance, as I'm sure this is a basic issue that stems from me not understanding something properly.

According to [the documentation](https://phantom-camera.dev/resource-types/tween), the tween resource can be used across multiple instances. However, even though I have local to scene disabled in the tween resource settings, when I change the duration for one instance it does not change for any of the others. How can I fix this issue?

**Comment by ramokz** [comment, marked as answer]
Don't think `local_to_scene` is meant for this, don't actually know what it's for :smile: 

But if you want to use a tween resource across multiple `PCam` instances, then there are two common ways of doing it.
Should mention that these approaches are the same for all resources within Godot, so is more of a general tip than an addon specific thing.

## Saved Resource File (Global and Local)
If you want to reuse the tween settings across multiple scenes, then this is the recommended approach, as you would save the resource as an external file that can be reused in any given scene.
1. Select the arrow of the `Tween Resource`, on the right-hand side, or right click
2. Select `Save As` near the bottom
3. Save it in a place that makes sense to you
4. Then in other `PCam` instances either:
  a. Drag the resource from the `FileSystem` into the `Tween Resource` slot
  b.  Or press on the arrow, on the right-hand side, and select `Load` and locate the saved resource you made in the above step

If you make any changes to the resource in a place where you've loaded it, it will change every instance where you have loaded the resource using the above steps

## Copy Resource (Local only)
If you want to only use the resource within the given scene, then you can do the below. This is only really a good idea if you know for sure that you will not need to use it in other scenes.
1. Select the arrow of the `Tween Resource`, on the right-hand side, or right click
2. Select `Copy` at the bottom
3. Go to another `PCam` instance
4. Click the arrow, on the right-hand side
5. Select `Paste` at the bottom

This basically copy a link from one resource to another, where changing one, changes another. If you want to override just one of them, you can click on the arrow, or right click, and select `Make Unique`. Note that this will break the link between that and where it was copied to / from. It does what it says on the tin.

**Reply by 2yellow4u** [reply]
Ah, I knew I was missing something fundamental! Thanks for the tip.
